core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/BenchmarkBase.scala

```diff
@@ -61,7 +65,10 @@ abstract class BenchmarkBase {
       file.createNewFile()
     }
     output = Some(new FileOutputStream(file))
-    runBenchmarkSuite(args)
+    runBenchmarkSuite(conf.iterations(),
+      conf.warmupIterations(),
```

Comment:
name `conf` is usually used with a configuration file/object.
Rename it to something more representative to arguments.
for example, `benchArgs`

Reply:
Updated to use benchArgs